_sass/bootstrap/_code.scss

```diff
@@ -8,6 +8,10 @@ code {
   a > & {
     color: inherit;
   }
+ 
+  span {
+    white-space: break-spaces;
```

Comment:
this is to eliminate horizontal scrolling on zoom. I think I can control this with a media query i.e. only wrap from a certain breakpoint and below. Thoughts?

Reply:
Are you sure that the accessibility guidelines don't have a special exception for boxes containing program code? 

For any GitHub comment or description containing a Markdown code block, they always display scrollbars instead of wrapping.  And take a look at these documentation pages from flagship products:
- https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.calendarblackoutdatescollection?view=windowsdesktop-5.0
- https://www.typescriptlang.org/docs/handbook/mixins.html
- https://docs.github.com/en/rest/reference/checks

Computer code is difficult to read if it wraps.  So I wonder if we might be misapplying the accessibility guidelines.  (?)

Otherwise, if you really think wrapping is indispensable, I'm okay with accepting this change.